cos/html/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
Where are the files under cos/html being referenced?  I don't see it being consumed so I'll suggest this folder to be removed.

Reply:
I believe we store the index.html in the cos instance: https://github.com/terraform-ibm-modules/terraform-ibm-static-website/blob/initial-version-code-dump/cos/main.tf#L43